ledger/sync/src/ledger_sync_service.rs

```diff
@@ -280,33 +279,33 @@ impl<L: Ledger, BC: BlockchainConnection + 'static, TF: TransactionsFetcher + 's
     /// Append safe blocks to the local ledger.
     fn append_safe_blocks(
         &mut self,
-        blocks_and_transactions: &[(Block, Vec<RedactedTx>)],
+        blocks_and_contents: &[(Block, BlockContents)],
     ) -> Result<(), LedgerSyncError> {
         log::info!(
             self.logger,
             "Appending {} blocks to ledger, which currently has {} blocks",
-            blocks_and_transactions.len(),
+            blocks_and_contents.len(),
             self.ledger
                 .num_blocks()
                 .expect("failed getting number of blocks"),
         );
         common::trace_time!(
             self.logger,
             "Appended {} blocks to ledger",
-            blocks_and_transactions.len()
+            blocks_and_contents.len()
         );
 
-        for (block, transactions) in blocks_and_transactions {
+        for (block, contents) in blocks_and_contents {
             {
                 let _timer = counters::APPEND_BLOCK_TIME.start_timer();
-                self.ledger.append_block(block, transactions, None)?;
+                self.ledger.append_block(block, contents, None)?;
             }
 
             // FIXME: MC-365 Move ledger counters into ledger_db
             counters::BLOCKS_WRITTEN_COUNT.inc();
-            counters::TX_WRITTEN_COUNT.inc_by(transactions.len() as i64);
+            // counters::TX_WRITTEN_COUNT.inc_by(transactions.len() as i64);
```

Comment:
Looks like we don't have TXO_WRITTEN_COUNT so this metric should probably be adapted to that. and the TX_IN_LEDGER one below (and wherever else it appears) removed.